python/peacock/tests/input_tab/InputTree/gold/fsp_test.i

```diff
@@ -1,27 +1,63 @@
+[Mesh]
```

Comment:
Oh OK. BTW - @gardnerru and @sapitts where asking me about the `--show-input` option to MOOSE a few weeks back. It would be useful to them and actual many others to eventually get that working properly (i.e. MOOSE spits out an equivalent working input file after processing the actual input file). This isn't a task I'm assigning you, just something to think about in the future. It would have really huge advantages when complicated custom Actions are used. One could process the input file to create a much larger machine generated input file with everything in it.